bitnami/etcd/templates/cronjob.yaml

```
@@ -30,6 +30,30 @@ spec:
           {{- if .Values.podSecurityContext.enabled }}
           securityContext: {{- omit .Values.podSecurityContext "enabled" | toYaml | nindent 12 }}
           {{- end }}
+          {{- if or .Values.initContainers .Values.volumePermissions.enabled }}
+          initContainers:
+            {{- if .Values.initContainers }}
+            {{- include "common.tplvalues.render" (dict "value" .Values.initContainers "context" $) | nindent 8 }}
+            {{- end }}
+            {{- if and .Values.volumePermissions.enabled (or .Values.podSecurityContext.enabled .Values.containerSecurityContext.enabled) }}
```

The `initContainers` parameter is intended for **etcd** pods exclusively:
```suggestion
          {{- if and .Values.volumePermissions.enabled (or .Values.podSecurityContext.enabled .Values.containerSecurityContext.enabled) }}
          initContainers:
```
Feel free to add a new parameter  (e.g. `disasterRecovery.initcontainers`) if you find it useful but do not reuse the existing one.

---

You're right, .Values.initContainers doesn't make sense in the context of the snapshotter cronjob, I removed it.